frontend/tests/factories/CoursesFactory.php

```diff
@@ -235,11 +235,14 @@ public static function addProblemsToAssignment(
     }
 
     /**
+     * @param array{course_alias: string} $courseData
      * @param \OmegaUp\DAO\VO\Identities[] $students
      * @param string[] $assignmentAliases
+     * @param array $problemAssignmentsMap
+     * @return array<string|null, array<string, int>>
```

Comment:
```suggestion
     * @return array<string, array<string, int>>
```

por qué ese `null`? según yo eso debería ser inválido O_o.

Reply:
Oh, no sé porque se lo puse, ya revisé y no se necesita.